src/PrestaShopBundle/Model/Product/AdminModelAdapter.php

```diff
@@ -329,7 +329,11 @@ public function getModelData($form_data, $isMultiShopContext = false)
 
         //map features
         if (!empty($form_data['features'])) {
-            foreach ($form_data['features'] as $dataFeature) {
+            foreach ($form_data['features'] as $dataFeature) {                
+                if (empty($dataFeature['feature']) {
```

Comment:
Ouch, you forgot to close a pair of parentheses, there is a syntax error here.

Reply:
Ouch. As you said. It's was not the base line I wrote, first. So fast...

I correct that !